Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/templates/* $KC_FOLDER/themes/"]
 # add integsoft version of keycloak jars
 RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/keycloak-modules/* $KC_FOLDER/lib/lib/main"]
 
-RUN /opt/keycloak/bin/kc.sh build --spi-email-sender-provider=aws-ses --spi-email-sender-provider-aws-ses-enabled=true --spi-email-sender-provider-aws-ses-region=$AWS_REGION
+RUN /opt/keycloak/bin/kc.sh build --spi-email-sender-provider=aws-ses --spi-email-sender-provider-aws-ses-enabled=true --spi-email-sender-provider-aws-ses-region=$AWS_REGION --features="token-exchange"
```

Comment:
question: should we also be enabling the `multi-site` feature to pick up the new health check? https://www.keycloak.org/docs/latest/release_notes/index.html#non-blocking-health-check-for-load-balancers

Reply:
This is not needed. Load balancing is done in ECS. And the health check we have already in place is good enough. For more information look at the infrastructure project: 
https://github.com/mbta/terraform-keycloak-sso/blob/d2686452bd3f0a72c803d900e839f6fe3b0a3de3/loadbalancer.tf#L64